src/main/resources/application.properties

```diff
@@ -1,12 +1,56 @@
 server.port=8081
 
-spring.datasource.url=jdbc:postgresql://postgres:5432/post_office_menagment
+spring.datasource.url=jdbc:postgresql://localhost:5432/post_office_menagment
```

Comment:
why localhost instead of docker image name?

Reply:
fix

Reply:
Can't see the fixed version